website/docs/configuration/environment_variables.md

```diff
@@ -6,50 +6,50 @@ sidebar_label: Environment Variables
+| **IMPORT_DIR**                         | The path of the directory for the import of data (e.g. TeslaFi)                                                                                                                                                                                                                                                  | ./import                                                                                                                                                            |
+| **TZ**                                 | Used to establish the local time zone, e.g. to use the local time in logs. See [List of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).                                                                                                                                   |                                                                                                                                                                     |
+| **DEFAULT_GEOFENCE**                   | The default GEOFENCE to send via GEOFENCE if car not in geofence. Overrides the default of "" which will delete any retained value.                                                                                                                                                                              | "" (no quotes)                                                                                                                                                      |
+| **TESLAFI_IMPORT_VEHICLE_ID**          | The default Vehicle ID used when importing from TeslaFi. If it is not defined and the import source does not contain the data, 1 is used.                                                                                                                                                                        |                                                                                                                                                                     |
+| **TESLA_API_HOST**                     | Hostname of the Tesla API                                                                                                                                                                                                                                                                                        | [https://owner-api.teslamotors.com](https://owner-api.teslamotors.com) (or for chinese: [https://owner-api.vn.cloud.tesla.cn](https://owner-api.vn.cloud.tesla.cn)) |
```

Comment:
This is bad practise imo, as it is hard to find any hidden issue, thats why I changed it to <URL> recently for URLs that are meant to be displayed directly and not as a link name via []()